tests/storage/storage.go

```diff
@@ -245,13 +245,15 @@ var _ = SIGDescribe("Storage", func() {
 				return libvmi.New(
 					libvmi.WithPersistentVolumeClaim("disk0", claimName),
 					libvmi.WithResourceMemory("256Mi"),
-					libvmi.WithRng())
```

Comment:
Out of curiosity, was this removed on purpose?

Reply:
Resolved offline. Short answer: Yes. This was removed temporarily until the cloud hypervisor supports it. It will be reintroduced later.